sqlglot/dialects/spark.py

```diff
@@ -87,6 +88,21 @@ class Parser(Hive.Parser):
             "SHUFFLE_REPLICATE_NL": lambda self: self._parse_join_hint("SHUFFLE_REPLICATE_NL"),
         }
 
+        CREATABLES = {
+            *Hive.Parser.CREATABLES,  # type: ignore
+            TokenType.COLUMNS,
```

Comment:
why did you add columns? i don't see how this is used?

Reply:
good point, I missed this -- this is indeed unneeded.